package/firmware/intel-microcode/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=intel-microcode
-PKG_VERSION:=20200616
-PKG_RELEASE:=1
+PKG_VERSION:=20210608
+PKG_RELEASE:=2
```

Comment:
What @jospezial meant was you should not be changing PKG_RELEASE from 1 to 2 snice you are changing the PKG_VERSION variable already. So PKG_RELEASE stays 1.

Reply:
Just to match the debian upstream